Learn-Information-Security-by-Building-a-Contact-Form/server060.js

```diff
@@ -0,0 +1,25 @@
+const express = require("express");
+const app = express();
+const helmet = require("helmet");
+// https://helmetjs.github.io/ - for more information on all packages and uses.
+
+app.use(helmet.dnsPrefetchControl());
+app.use(helmet.hidePoweredBy());
+
+app.use(helmet.ieNoOpen());
+// Sets the'X-Download-Options' to prevent IE from downloading.
+
+app.use(helmet.noSniff());
```

Comment:
`app.use(helmet.noSniff());`, `app.use(helmet.xssFilter());`, and `app.use(helmet.frameguard());` could all be split into separate steps, each explaining why the default header setting can be dangerous.